best_practices/templates.rst

```diff
@@ -51,6 +51,10 @@ Another advantage is that centralizing your templates simplifies the work
 of your designers. They don't need to look for templates in lots of directories
 scattered through lots of bundles.
 
+.. best-practice::
+
+    Use lowercased snake_case for directory and template names.
```

Comment:
ping @javiereguiluz I need your input on upper/lower case template directories here - this will affect a lot of different parts. Thanks :)